posthog/clickhouse/migrations/0132_add_max_inserted_at_to_raw_sessions.py

```diff
@@ -0,0 +1,27 @@
+from posthog.clickhouse.client.connection import NodeRole
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.raw_sessions.migrations import (
+    BASE_RAW_SESSIONS_ADD_MAX_INSERTED_AT_COLUMN_SQL,
+    DISTRIBUTED_RAW_SESSIONS_ADD_MAX_INSERTED_AT_COLUMN_SQL,
+    WRITABLE_RAW_SESSIONS_ADD_MAX_INSERTED_AT_COLUMN_SQL,
+)
+from posthog.models.raw_sessions.sql import (
+    DROP_RAW_SESSION_MATERIALIZED_VIEW_SQL,
+    RAW_SESSIONS_TABLE_MV_SQL,
+    RAW_SESSIONS_CREATE_OR_REPLACE_VIEW_SQL,
+)
+
+operations = [
+    # drop the mv, so we are no longer receiving events from the sessions table
+    run_sql_with_exceptions(DROP_RAW_SESSION_MATERIALIZED_VIEW_SQL()),
+    # now we can alter the target tables
+    run_sql_with_exceptions(BASE_RAW_SESSIONS_ADD_MAX_INSERTED_AT_COLUMN_SQL()),
```

Comment:
why not node_role=NodeRole.ALL here?

Reply:
I just copied this from a previous migration. I'm guessing things have changed since then, but wanted to get *something* up to get feedback on

Reply:
Is the raw_sessions table available on coordinator? I have impression it's materialized only on data nodes.